### PR TITLE
Add source parameter support

### PR DESCRIPTION
**Pull request by lgryn**
For my current project, I need to return different fields sets on searches. My proposal add simple changes to allow index-helper users specify fields what they need.

### CODE REVIEW

**Comment by Nexucis** [comment]
I just closed and reopened this PL to trigger circle-ci. The build from a fork was not activated, sorry for that

File: src/Nexucis/Elasticsearch/Helper/Nodowntime/IndexHelper.php [hunk 1]
```diff
@@ -457,7 +458,11 @@ public function searchDocuments($alias, $query = null, $type = null, $from = 0,
         );
 
         if (is_array($query)) {
-            $params['body'] = array('query' => $query);
+            $params['body']['query'] = $query;
+        }
+
+        if (is_array($source)) {
+            $params['body']['_source'] = $source;
```
Comment (Nexucis) [inline src/Nexucis/Elasticsearch/Helper/Nodowntime/IndexHelper.php]: as you can see on the [php documentation](https://github.com/elastic/elasticsearch-php/blob/master/src/Elasticsearch/Client.php#L921) , the `_source` parameter is not something that shall be in the body but directly in the root of the parameter  : 

```php
$params['_source'] = $source
```

Comment (lgryn) [reply src/Nexucis/Elasticsearch/Helper/Nodowntime/IndexHelper.php]: Hey. Yes, it's true. I miss that because it works in both ways.

**Comment by Nexucis** [comment]
Thanks for the changing and your contribution :)

**Pull request merged**